types/@gavant/gavant-ember-validations/input-validator.d.ts

```diff
@@ -0,0 +1,14 @@
+import FormValidatorComponent from '@gavant/glint-template-types/types/@gavant/gavant-ember-validations/form-validator';
+import Component from '@glint/environment-ember-loose/ember-component';
+
+export interface InputValidatorSignature {
+    Args: {
+        errors: string | string[];
```

Comment:
It seems like errors must be an array? https://github.com/Gavant/gavant-ember-validations/blob/master/addon/components/input-validator.ts#L31